browser/ui/views/tabs/BUILD.gn

```diff
@@ -32,3 +32,23 @@ source_set("browser_tests") {
     ]
   }
 }
+
+source_set("unit_tests") {
+  testonly = true
+
+  sources = [
+    "//chrome/browser/ui/views/tabs/fake_tab_slot_controller.cc",
```

Comment:
why are these included here?

Reply:
Unfortunately, these two files are only included in upstream `unit_tests` target.